internal/api/v2/documents.go

```diff
@@ -968,9 +968,9 @@ func updateRecentlyViewedDocs(
 		docs = append(docs, dd)
 	}
 
-	// Trim recently viewed documents to a length of 5.
-	if len(docs) > 10 {
-		docs = docs[:10]
+	// Trim recently viewed documents to a length of 11.
```

Comment:
Did we resolve the sorting issues we saw with sorting the recently viewed docs?

Reply:
There functionally aren't any issues with recently viewed docs, but there's room for improvement here - I'm going to tackle this soon!